pom.xml

```diff
@@ -886,6 +887,28 @@
                 <version>5.7.22-1</version>
             </dependency>
 
+            <dependency>
```

Comment:
I would define that in hive only, as no other module is using this.

Reply:
It's fine either way. Our general rule is to always put dependences in the root POM, but putting very connector specific things in the connector POMs is reasonable as well, especially as we add more.